frontend/lib/json/event.dart

```diff
@@ -83,16 +56,16 @@ class EventInfo {
       eventName: json['event_name'],
       description: json['description'],
       hostId: json['host_id'],
-      isPublic: json['isPublic'], 
```

Comment:
Change the keys to proper JSON names

Reply:
great! should be able to retrieve data correctly